mrbelvedereci/plan/views.py

```diff
@@ -6,10 +6,18 @@
 from mrbelvedereci.build.utils import view_queryset
 from mrbelvedereci.plan.models import Plan
 from mrbelvedereci.plan.forms import RunPlanForm
+from mrbelvedereci.repository.models import Repository
 
 
-def plan_list(request, plan_id):
-    return ''
+def plan_list(request):
+    if request.user.is_staff:
+        plans = Plan.objects.all()
+    else:
+        plans = Plan.objects.filter(public=True)
```

Comment:
Confirming what this does for SAL etc....

SAL will still be private at the *REPO* level, but the plan will be public? Logged out users will be able to go to Feature Test plan and see NPSP and HEDA on it, but not SAL?

Reply:
Good point, but I think in your example wouldn't a logged out user be able to see a SAL plan as long as it was public? I filed this one a while ago and I still think we should implement it: https://github.com/SalesforceFoundation/mrbelvedereci/issues/46 – Even if it doesn't sync to github I still think we should just control public/private at the repo level, but we should continue the debate in that issue. As far as this here is concerned, I think it's consistent with our models, so that if a plan is public but one of the repos it runs against is private, we would show the plan anyway.

Reply:
A logged out user will not see any private plans in this view, and public plans with an associated private repo would show that repo in their list, which I think we don't want. So I'll change the repos list to take public/private into account.

Reply:
Addressed in https://github.com/SalesforceFoundation/mrbelvedereci/pull/147/commits/a7aa974593643747d2218b3ff3be4533e88d92a1